src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/generated.py

```diff
@@ -126,3 +129,55 @@
+cli_storage_data_plane_command('storage table policy delete', delete_acl_policy, factory)
+cli_storage_data_plane_command('storage table policy show', get_acl_policy, factory)
+cli_storage_data_plane_command('storage table policy list', list_acl_policies, factory, simple_output_query=transform_acl_list_output)
+cli_storage_data_plane_command('storage table policy update', set_acl_policy, factory)
+cli_storage_data_plane_command('storage table batch commit', TableService.commit_batch, factory)
+cli_storage_data_plane_command('storage table batch create', TableService.batch, factory)
+
+# table entity commands
+cli_storage_data_plane_command('storage entity query', TableService.query_entities, factory)
+cli_storage_data_plane_command('storage entity show', TableService.get_entity, factory)
+cli_storage_data_plane_command('storage entity insert', insert_table_entity, factory)
+cli_storage_data_plane_command('storage entity replace', TableService.update_entity, factory)
+cli_storage_data_plane_command('storage entity merge', TableService.merge_entity, factory)
+cli_storage_data_plane_command('storage entity delete', TableService.delete_entity, factory)
+
+# queue commands
+factory = queue_data_service_factory
+cli_storage_data_plane_command('storage queue generate-sas', QueueService.generate_queue_shared_access_signature, factory)
+cli_storage_data_plane_command('storage queue stats', QueueService.get_queue_service_stats, factory)
+cli_storage_data_plane_command('storage queue service-properties show', QueueService.get_queue_service_properties, factory)
+cli_storage_data_plane_command('storage queue service-properties update', QueueService.set_queue_service_properties, factory)
+cli_storage_data_plane_command('storage queue list', QueueService.list_queues, factory, simple_output_query='items[*].{Name:name}')
+cli_storage_data_plane_command('storage queue create', QueueService.create_queue, factory)
+cli_storage_data_plane_command('storage queue delete', QueueService.delete_queue, factory)
+cli_storage_data_plane_command('storage queue metadata show', QueueService.get_queue_metadata, factory)
+cli_storage_data_plane_command('storage queue metadata update', QueueService.set_queue_metadata, factory)
+cli_storage_data_plane_command('storage queue exists', QueueService.exists, factory)
+cli_storage_data_plane_command('storage queue policy create', create_acl_policy, factory)
+cli_storage_data_plane_command('storage queue policy delete', delete_acl_policy, factory)
+cli_storage_data_plane_command('storage queue policy show', get_acl_policy, factory)
+cli_storage_data_plane_command('storage queue policy list', list_acl_policies, factory, simple_output_query=transform_acl_list_output)
+cli_storage_data_plane_command('storage queue policy update', set_acl_policy, factory)
+
+# queue message commands
+cli_storage_data_plane_command('storage message put', QueueService.put_message, factory)
```

Comment:
Same as above.

---
In reply to: [74452937](https://github.com/Azure/azure-cli/pull/681#discussion_r74452937) [](ancestors = 74452937)